.github/workflows/build-and-deploy-new-acct.yml

```diff
@@ -0,0 +1,74 @@
+# temporary workflow to start deploying to the new prod account.
+name: Build and deploy new account
+on:
+  push:
+    branches:
+      - master
+jobs:
+  buildSite:
+    env:
+      GOPATH: ${{ github.workspace }}/go
+    name: Install deps and build site
+    runs-on: buildjet-4vcpu-ubuntu-2004
```

Comment:
I didn't realize we were using buildjet for this.

We do have native 4core github runners now, which IIRC are a tad more expensive, but mean we don't need to spread our keys onto another service (I was in the middle of trying to get buildjet's SOC2 when the github runners became available)